### ISSUE TITLE
TypeNotSupportedException when generating schema

### ISSUE THREAD

**Issue by VincentHoang** [ExpediaGroup/graphql-kotlin#710]
Hello,

After adding the following:

```
data class Widget(val id: Int, val value: String)

// Mark the class as a Spring GraphQL Query
@Component
class WidgetService : Query {
    fun widgetById(id: Int): Widget? {
        return null
    }
}

@SpringBootApplication
class GraphqlApplication

fun main(args: Array<String>) {
    runApplication<GraphqlApplication>(*args)
}

```
I'm getting this error:
> Caused by: com.expediagroup.graphql.exceptions.TypeNotSupportedException: Cannot convert com.example.graphql.query.Widget? since it is not a valid GraphQL type or outside the supported packages ["com.example.graphql"]

I'm following a very basic example from the GraphQL Kotlin documentation: https://expediagroup.github.io/graphql-kotlin/docs/spring-server/spring-overview

It works if I replaced` Widget? `with `String?`

I'm using graphql-kotlin-spring-server **version 2.1.1**

Any ideas? Sorry if I've missed anything obvious. 

Thanks

**Comment by smyrick** [comment]
@VincentHoang The first step is to specify the configuration. At a minimum you need to specify the packages to scan for the GraphQL schema classes

```yml
graphql:
  packages:
    - "com.your.package"
```

See the `Configuration` section in the docs you linked to: https://expediagroup.github.io/graphql-kotlin/docs/spring-server/spring-overview#configuration

**Comment by VincentHoang** [comment]
Thanks, but I've done that already

**Comment by smyrick** [comment]
I would double check with our example app setup to make sure you have the same dependencies: https://github.com/ExpediaGroup/graphql-kotlin/tree/master/examples/spring

Can you provide a link to your repo? I don't see anything else that stands out

**Comment by VincentHoang** [comment]
I've created a new repo. Please have a look: https://github.com/VincentHoang/graqhql

I created the project using Spring Initializr, then added the graphql-kotlin-spring-server dependency simply by following this: https://expediagroup.github.io/graphql-kotlin/docs/spring-server/spring-overview

Somehow it can't convert the data class Widget

Thanks

**Comment by smyrick** [comment]
@VincentHoang I cloned your repo and started the application in debug mode.

I put a break point in `TypesCache` where we throw the exception and was able to see that the error was due to the supported packages being `"com.example.graphql"` instead of `com.example.graphql`.

If you updated your application.properties file to this it then works

```properties
graphql.packages=com.example.graphql
```

This is due to the different way that properties files are handled vs yaml files.
